heartbeat/heartbeat.go

```diff
@@ -140,6 +140,8 @@ func (hb *HeartBeat) Run(ctx context.Context, wg *sync.WaitGroup) error {
 func heartBeatWorker(ctx context.Context, wg *sync.WaitGroup, sub *Subject, nc *nats.Conn, js nats.JetStreamContext, replicatorName string, paused *atomic.Bool, log *logrus.Entry) {
 	defer wg.Done()
 
+	log.Infof("Starting heartbeat on subject: %s", sub.name)
```

Comment:
we have a label for the subject, so thats redundant, lets instead say `Starting heartbeat with interval %v`